bedrock/mozorg/templates/mozorg/MPL/licence-policy.html

```diff
@@ -0,0 +1,95 @@
+            <li>You must consult <a href='#Filing_Bugs'>the licensing team</a> before importing third-party code which is not under the MPL or Apache 2.0.</li>
+        </ul>
+    </section>
+
+    <section>
+        <h2 id="Types_of_Code">Types of Code</h2>
+        <p><b>Mozilla Code</b> is code in Mozilla Repositories which originated with the Mozilla Project. Code which is not Mozilla Code is <b>Third Party Code</b>.</p> 
+        <p><b>Product Code</b> means all files any part of which is included in nightly binaries of the Mozilla family of Internet clients - Firefox, Thunderbird, SeaMonkey and Camino - plus the Gecko parts of Boot2Gecko. Note that this definition excludes the code of tools used merely to build these products, such as system libraries.</p> 
+        <p><b>Test Code</b> means test cases for automated test frameworks (but not the frameworks themselves).</p>
+    </section>
+
+    <section>
+        <h2 id="Licensing_of_Mozilla_Code">Licensing of Mozilla Code</h2>
+        <p>New files in, modifications to, or code designed to integrate with or build on an existing, consistently-licensed area of Mozilla Code should be under the same license as the existing code. This means that modifications to Product Code will be MPL 2.</p>
+        <p>New projects which are Mozilla Code may choose either the MPL 2.0 or the Apache License 2.0. No other license is acceptable, because Mozilla is committed to using modern open source licenses with patent grant clauses. The licensing team recommends MPL 2.0 for client-side code; please consult the team before going against this recommendation.</p>
+        <p>Trivial bits of Mozilla Code, such as testcases or snippets of code used in documentation, should be put in the public domain in order to facilitate re-use. To do that, apply the <a href="http://creativecommons.org/publicdomain/zero/1.0/">Creative Commons Public Domain Dedication</a> by using the following boilerplate: </p>
+        <blockquote>
+        <pre>
+        Any copyright is dedicated to the Public Domain.
+        http://creativecommons.org/publicdomain/zero/1.0/
+        </pre>
+        </blockquote> 
+        <p>Note that, for historical reasons, some trivial Support Code, such as <a href="https://developer.mozilla.org/Project:Copyrights">old code samples in developer.mozilla.org</a>, may be under the MIT license.</p> <p><b>PD Test Code</b> is Test Code which is Mozilla Code, which does not carry an explicit license header, and which was either committed to the Mozilla repository on or after 23rd September 2014, or was committed before that date but all contributors up to that date were Mozilla employees, contractors or interns. PD Test Code is made available under the <a href="http://creativecommons.org/publicdomain/zero/1.0/">Creative Commons Public Domain Dedication</a>. Test Code which has not been demonstrated to be PD Test Code should be considered to be under the MPL 2.</p>
```

Comment:
This link is dead (both in the ported page and the old SVN page). Maybe we should ask on the bug if the "such as old code samples in developer.mozilla.org" phrase can be removed, or if there is a more appropriate new link.